css/style.css

```diff
@@ -972,9 +990,15 @@ button.button-inline:hover {
 		display: none;
 	}
 
+	#innerBoard {
+		display: flex;
+	}
+
 	.stack {
 		width: 320px;
+		min-width: 320px;
 		display: inline-block;
+		position: relative;
```

Comment:
not required - since no positioning values (e.g. `top`) are applied to this class.